Quests/Buster Quests/quest_00300170.csv

```diff
@@ -1,11 +1,11 @@
 Title#0,"""Battle the Devil Castle"""
-Explanation#0,"""異世界オメガに存在する魔神城が活性化。<br>かの世界で戦乱が起こっています。<br>オメガへ向かい、魔神城を撃破しましょう。<br>魔神城の結界を突破するための兵器<br>「破界鎚」が鍵になる戦いですよ！<br>頑張りましょう、守護輝士！<br>――――――――――――――――――――<br>制限時間の経過でクエスト終了<br>敵の戦力ゲージを０にするか<br>制限時間経過時に残量が上回っていれば勝利"""
-StoryExplanation#0,"""After a face to face meeting with Alisa,<br>the Sword Maiden, Harriet finds herself<br>embroiled in another battle, as Lutz<br>summon a Devil Castle in front of the camp!"""
+Explanation#0,"""A Devil Castle has appeared in Omega.<br>War has broken out.<br>Travel to Omega and destroy the Devil Castle.<br>The \""Buster Piles\"" are the key to<br>breaking through the Devil Castle!<br>Let's get to it,  Guardians!<br>――――――――――――――――――――<br>Quest ends when time limit expires.<br>If the enemy gauge hits 0, or your gauge is<br>higher at the end of the quest, you win."""
```

Comment:
Don't need to translate this bit, as it's just SEGA reusing the file, it uses the `StoryExplanation#0` instead.

Reply:
does it not use this line for the people who join it as a free quest?